gatsby_fp/src/components/Layout.js

```diff
@@ -27,6 +24,10 @@ export default function Layout({ children, data }) {
     }
   }
 
+  //Conditionally declare sanitizedData in case API returns no content
```

Comment:
As a point of style, we prefer a space between comment syntax and the comment itself.